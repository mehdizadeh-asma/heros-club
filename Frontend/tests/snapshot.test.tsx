import { render } from '@testing-library/react'
import Home from '@/pages/index'
import Dashboard from '@/pages/admin'

it('renders homepage unchanged', () => {
  const { container } = render(<Home />)
  expect(container).toMatchSnapshot()
})

it('renders admin unchanged', () => {
  const { container } = render(<Dashboard />)
  expect(container).toMatchSnapshot()
})

