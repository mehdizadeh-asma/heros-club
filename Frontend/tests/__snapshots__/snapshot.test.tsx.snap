// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders admin unchanged 1`] = `
<div>
  <div>
    <main>
      <div
        class="container pt-0 m bg-dark text-white"
      />
    </main>
    <footer
      class="footer"
    />
  </div>
</div>
`;

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="container"
  >
    <main>
      <a
        href="/admin"
      >
        go to admin dashborad
      </a>
    </main>
    <footer
      class="footer"
    />
  </div>
</div>
`;
